content/posts/02_staff_expectations/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "What is expected from staff engineers"
+date: 2022-12-18
+tags: [
+    "career", "growth"
+]
+---
+
+It's relatively easy to explain expectations for junior, mid, and even senior engineers. But it starts to get much harder at staff+ 
+positions. Career progression broadens the scope of work andresponsibilities. The environment and problems get more ambiguous. It's not easy
+to define a clear and measurable set of expectations, because staff roles also differ between companies and
+their actual needs. They can also vary between different [staff engineer archetypes](https://staffeng.com/guides/staff-archetypes).
+
+In the article, we will focus mostly on the **Tech Lead** archetype, which is the most popular in the industry. We will present some 
+expectations for staff engineers based on our experience. The list is not complete and there are many very important skills and 
+traits a staff engineer should have. We wanted to describe those that are fundamental regardless of the specific situation.
+
+# So what are those expectations?
+
+## Get things done
+![](./done.jpg)
+
+This might as well be applied to the earlier positions. But for staff engineers, it gets even more important. Staff engineers have to get 
+their own work done and make sure others can do it too. They should help other engineers and teams manage and prioritize complex projects. 
+They should act as a "glue" to ensure engineering teams are working and delivering smoothly. They should observe and eliminates the developer
```

Comment:
We've got 3 repetitions of `They should` is it intentional?

Reply:
It seems it doesn't work well. I rewrote this paragraph using ChatGPT. :D